cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx

```diff
@@ -1,20 +1,38 @@
 // React imports
-import React from 'react';
+import React, {useContext, useEffect} from 'react';
 import { Routes, Route } from 'react-router-dom';
 
 // App imports
 import HeaderComponent from './shared/components/header/header.component';
-import SideNavigationComponent from './shared/components/side-navigation/side-navigation.component';
+import SideNavigationComponent from './shared/components/side-navigation/SideNavigation.component';
 import GetStartedComponent from './views/getting-started/getting-started.component';
 import VSphere from './components/VSphere';
 import WelcomeComponent from './views/welcome/welcome.component';
+import ProgressComponent from './views/temp/progress.component';
+import { APP_ENV_CHANGE } from './state-management/actions/app.actions';
+import { Store } from './state-management/stores/store';
 
 function App(this: any) {
```

Comment:
what is the `this: any`? do we need it?

Reply:
that's weird, I looked right over it. I think it was maybe part of the seed project that I started from. I removed it and the app loads fine.